### PR TITLE
Add release to runtime pod and delete with release

### PR DESCRIPTION
**Pull request by ningyougang**
## Description

If want to deploy two helm release openwhisk cluster,  when the second helm release cluster's invokerN pod is restarting, it will delete the first helm release's invokerN runtime pods.

Because when invokerN pod is starting, it will delete  runtime pods with label(invoker=invokerN) only. e.g.
https://github.com/apache/openwhisk/blob/master/core/invoker/src/main/scala/org/apache/openwhisk/core/containerpool/kubernetes/KubernetesContainerFactory.scala#L65

in order to avoid `delete other helm release invokerN's runtime pod`, it is better to add helm release to invoker pod, and when invoker pod is starting, add label(release=xxxxxx) together with label(invoker=invokerN) to delete runtime pods.

Another brother pr: https://github.com/apache/openwhisk-deploy-kube/pull/639

## Related issue and scope

- [ ] I opened an issue to propose and discuss this change (#????)

## My changes affect the following components

- [ ] API
- [ ] Controller
- [ ] Message Bus (e.g., Kafka)
- [ ] Loadbalancer
- [x] Invoker
- [ ] Intrinsic actions (e.g., sequences, conductors)
- [ ] Data stores (e.g., CouchDB)
- [ ] Tests
- [ ] Deployment
- [ ] CLI
- [ ] General tooling
- [ ] Documentation

## Types of changes

- [x] Bug fix (generally a non-breaking change which closes an issue).
- [ ] Enhancement or new feature (adds new functionality).
- [ ] Breaking change (a bug fix or enhancement which changes existing behavior).

## Checklist:

- [x] I signed an [Apache CLA](https://github.com/apache/openwhisk/blob/master/CONTRIBUTING.md).
- [x] I reviewed the [style guides](https://github.com/apache/openwhisk/wiki/Contributing:-Git-guidelines#code-readiness) and followed the recommendations (Travis CI will check :).
- [ ] I added tests to cover my changes.
- [ ] My changes require further changes to the documentation.
- [ ] I updated the documentation where necessary.

### CODE REVIEW

**Comment by dgrove-oss** [comment]
Putting the two deployments in separate Kubernetes namespaces should also avoid the problem (I think).

**Comment by ningyougang** [comment]
@dgrove-oss ,yes, in two kubernateds namespace can avoid the problem.
but some users may deploy two deployments in same namespaces

hm..i think add a one more lable(release=xxxx) it is more carefully.

File: tests/src/test/resources/application.conf.j2 [hunk 1]
```diff
@@ -100,6 +100,8 @@ whisk {
     elasticsearch {
         docker-image = "{{ elasticsearch.docker_image | default('docker.elastic.co/elasticsearch/elasticsearch:' ~ elasticsearch.version ) }}"
     }
+
+    helm.release = "release"
```
Comment (style95) [inline tests/src/test/resources/application.conf.j2:104]: Just for clarity, how is this value overridden?

**Review by style95 — APPROVED** [review]
LGTM

**Pull request merged**